src/metabase/api/search.clj

```diff
@@ -151,7 +152,7 @@
 (s/defn ^:private from-clause-for-model :- [(s/one [(s/one s/Keyword "table name") (s/one s/Keyword "alias")]
                                                    "from clause")]
   [model :- SearchableModel]
-  (let [db-model (get search-config/model-to-db-model model)]
+  (let [db-model (get search-config/model-to-db-model-t1 model)]
```

Comment:
You should be able to use `search-config/model-to-db-model` here. `t2/table-name` will call `name` on a keyword.

Reply:
per Cam's [message](https://metaboat.slack.com/archives/CKZEMT1MJ/p1675879200978579), looks like the preferable way is to use keyword with toucan2?

Reply:
I'm unsure what part of Cam's message you're talking about, but yes I'm advocating for using keywords and toucan2

Reply:
As in, you shouldn't need `model-to-db-model-t1` anymore. Just use the toucan2 model keywords